components/dashboard/src/projects/new-project/NewProjectCreateFromURL.tsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { FC, useCallback, useMemo } from "react";
+import isURL from "validator/lib/isURL";
+import { CreateProjectArgs } from "../../data/projects/create-project-mutation";
+import { Subheading } from "../../components/typography/headings";
+import { Button } from "../../components/Button";
+import { useToast } from "../../components/toasts/Toasts";
+
+type Props = {
+    repoSearchFilter: string;
+    isCreating: boolean;
+    onCreateProject: (args: CreateProjectArgs) => void;
+};
+export const NewProjectCreateFromURL: FC<Props> = ({ repoSearchFilter, isCreating, onCreateProject }) => {
+    const { toast } = useToast();
+    const showCreateFromURL = useMemo(() => {
+        // TODO: Only accounts for https urls, need to account for ssh clone urls too?
+        const looksLikeURL = isURL(repoSearchFilter, {
+            require_protocol: true,
+            protocols: ["https"],
+        });
+
+        const hasTrailingGit = /\.git$/.test(repoSearchFilter);
+
+        return looksLikeURL && hasTrailingGit;
+    }, [repoSearchFilter]);
+
+    const normalizedURL = useMemo(() => {
+        let url = repoSearchFilter.toLowerCase().trim();
+
+        return url;
+    }, [repoSearchFilter]);
+
+    const handleCreate = useCallback(() => {
+        let name = "";
+        let slug = "";
+
+        try {
+            // try and parse the url for owner/repo path parts
+            console.log("url: ", normalizedURL.substring(0, normalizedURL.length - 4));
+            const [owner, repo] = new URL(normalizedURL.substring(0, normalizedURL.length - 4)).pathname.split("/");
```

Comment:
This split produces an array like this: 
```ts
[ "", "gitpod-io", "workspace-images" ]
```

So we should filter out the first item of the array. Here's 2 different ways I can think of to do that:

```suggestion
            const [_, owner, repo] = new URL(normalizedURL.substring(0, normalizedURL.length - 4)).pathname.split("/");
```

```suggestion
            const [owner, repo] = new URL(normalizedURL.substring(0, normalizedURL.length - 4)).pathname.split("/").filter(Boolean);
```

Also, for better readability, I'd be explicit with what the substringing is for
```suggestion
            const [owner, repo] = new URL(normalizedURL.substring(0, normalizedURL.length - ".git".length)).pathname.split("/");
```